make/Hsdis.gmk

```diff
@@ -129,13 +151,18 @@ TARGETS += build
 INSTALLED_HSDIS_NAME := hsdis-$(OPENJDK_TARGET_CPU_LEGACY_LIB)$(SHARED_LIBRARY_SUFFIX)
 
 INSTALLED_HSDIS := $(INSTALLED_HSDIS_DIR)/$(INSTALLED_HSDIS_NAME)
+INSTALLED_HSDIS_IMAGE := $(IMAGE_HSDIS_DIR)/$(INSTALLED_HSDIS_NAME)
 
 $(INSTALLED_HSDIS): $(BUILD_HSDIS_TARGET)
-	$(call LogWarn, NOTE: The resulting build might not be redistributable. Seek legal advice before distibuting.)
+ifeq ($(HSDIS_BACKEND), binutils)
```

Comment:
Don't we usually indent conditionals to the recipe level using spaces?

Reply:
Yes we do. Embarrassing. 😊